components/App.vue

```diff
@@ -344,12 +343,9 @@ async function getEventSources() {
     '/api/events/wix',
     */
   ];
-  const clientHeaders = {
-    'Cache-Control': `max-age=${clientCacheMaxAgeSeconds}, stale-while-revalidate=${clientStaleWhileInvalidateSeconds}`,
-  };
   // This is to preventing the UI changes from each fetch result to cause more fetches to occur.,
   Promise.allSettled(endpoints.map(async (endpoint) => {
-    const { data: response } = await useLazyFetch(endpoint, { headers: clientHeaders });
+    const { data: response } = await useLazyFetch(endpoint);
```

Comment:
Example of removing manually set headers since caching should be handled by nuxt.config.ts